servers/rendering/renderer_rd/forward_clustered/render_forward_clustered.h

```diff
@@ -287,6 +287,9 @@ class RenderForwardClustered : public RendererSceneRenderRD {
 			float reflection_multiplier;
 
 			uint32_t pancake_shadows;
+
+			float color_scale;
```

Comment:
This is not really needed in clustered.

Reply:
The problem is that the clustered and mobile renderer share code, one needs it, the other doesn't...

Reply:
Now solved with a constant